src/types/date-time.ts

```diff
@@ -1,9 +1,10 @@
-import {STRules} from '../rules';
 import {StrongType, makeStrong} from '../strong-type';
 
+import {STRules} from '../rules';
+
 export type StrongDateTime = StrongType<string>;
 
-export function makeDateTime(fallback: string, initial?: string | null): StrongDateTime {
+export function makeDateTime(fallback: string, initial: string | null | undefined): StrongDateTime {
```

Comment:
- [ ] Change `| undefined` type on `initial` to use the optional operator instead `?`.